Duckling/Quantity/EN/Rules.hs

```diff
@@ -33,22 +33,17 @@ import qualified Duckling.Quantity.Types as TQuantity
 quantities :: [(Text, String, TQuantity.Unit)]
 quantities =
   [ ("<quantity> cups", "(cups?)", TQuantity.Cup)
-  , ("<quantity> grams", "(((m(illi)?)|(k(ilo)?))?g(ram)?s?)", TQuantity.Gram)
+  , ("<quantity> pieces", "(pieces?)", TQuantity.Piece)
+  , ("<quantity> milligrams", "(m(illi)?g(ram)?s?)", TQuantity.Milligram)
+  , ("<quantity> grams", "(g(ram)?s?)", TQuantity.Gram)
+  , ("<quantity> kilograms", "(k(ilo)?g(ram)?s?)", TQuantity.Kilogram)
   , ("<quantity> lb", "((lb|pound)s?)", TQuantity.Pound)
   , ("<quantity> oz", "((ounces?)|oz)", TQuantity.Ounce)
+  , ("<quantity> tonnes", "(ton(ne)?s?)", TQuantity.Tonne)
```

Comment:
the abbreviations for Tonne and Pieces will probably not work

Reply:
No, they won't work right now. I will add them.